docs/pipelines/tasks/utility/batch-script.md

```diff
@@ -108,7 +108,7 @@ This task is open source [on GitHub](https://github.com/Microsoft/vsts-tasks). F
 
 ### Where can I learn about batch files?
 
-[Using batch files](http://www.microsoft.com/resources/documentation/windows/xp/all/proddocs/en-us/batch.mspx?mfr=true)
+[Using batch files](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands)

```

Comment:
Should probably be `/windows-server/administration/windows-commands/windows-commands` since it is a link on docs.microsoft.com

Reply:
Thanks! But as far as I know, many other docs pages in MSFT Docs that refers to other departments/topics outside the scope of their own repo are having links as full absolute links, instead of relative links. And having relative links in the MD will point to not existed page for reviewer.

Reply:
I asked a related on about this before https://github.com/MicrosoftDocs/feedback/issues/587, the `en-us` part should be removed even if you keep the domain
https://docs.microsoft.com/en-us/contribute/how-to-write-links